diff --git a/plugin/server/tcp_server/tcp_server.go b/plugin/server/tcp_server/tcp_server.go
--- a/plugin/server/tcp_server/tcp_server.go
+++ b/plugin/server/tcp_server/tcp_server.go
@@ -76,7 +76,9 @@ func StartServer(bp *coremain.BP, args *Args) (*TcpServer, error) {
 	// Init tls
 	var tc *tls.Config
 	if len(args.Key)+len(args.Cert) > 0 {
-		tc = new(tls.Config)
+		tc = &tls.Config{
+			SessionTicketsDisabled: true,
+		}
 		if err := server.LoadCert(tc, args.Cert, args.Key); err != nil {
 			return nil, fmt.Errorf("failed to read tls cert, %w", err)
 		}
